code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -158,6 +213,23 @@ components:
           description: Hashed phone number. SHA-256 (in hexadecimal representation) of the mobile phone number in 'E164 with +' format
           type: string
           example: 32f67ab4e4312618b09cd23ed8ce41b13e095fe52b73b2e8da8ef49830e50dba
+    ueId:
```

Comment:
About the parameters name (ipv4Addr, ipv6Addr, uePort), they are voting in other APIs working groups. We'll align them with the result that comes up from there.